x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/footer.tsx

```diff
@@ -64,6 +64,7 @@ export const EventDetailsFooterComponent = React.memo(
       [detailsData]
     );
 
+    // TODO prolly a bug
```

Comment:
when this PR is taken out of `draft`, consider updating this with a link to the issue if the final version doesn't include a fix